core/src/main/java/org/apache/accumulo/core/clientImpl/TableOperationsImpl.java

```diff
@@ -1477,6 +1479,47 @@ public Text getMaxRow(String tableName, Authorizations auths, Text startRow,
     return finalUsages;
   }
 
+  /**
+   * Search multiple directories for exportMetadata.zip, the control file used for the importable
+   * command.
+   *
+   * @param context
+   *          used to obtain filesystem based on configuration
+   * @param importDirs
+   *          the list of directories to search.
+   * @return the Path representing the location of the file.
+   * @throws AccumuloException
+   *           if zero or more than one copy of the exportMetadata.zip file are found in the
+   *           directories provided.
+   */
+  public static Path findExportFile(ClientContext context, String[] importDirs)
+      throws AccumuloException {
+    LinkedHashSet<Path> exportFiles = new LinkedHashSet<>();
+    for (String importDir : importDirs) {
+      Path exportFilePath = null;
+      try {
+        FileSystem fs = new Path(importDir).getFileSystem(context.getHadoopConf());
+        exportFilePath = new Path(importDir, Constants.EXPORT_FILE);
+        if (fs.exists(exportFilePath)) {
+          exportFiles.add(exportFilePath);
+        }
+      } catch (IOException ioe) {
+        log.warn("Non-Fatal IOException reading export file: {}", exportFilePath, ioe);
```

Comment:
I think I would be confused by this log message. How do you know it's non-fatal? And, what level of concern should I have as a user to address it?